src/api/procedures/__tests__/utils.ts

```diff
@@ -843,8 +841,8 @@ describe('authorization request validations', () => {
     });
   });
 
-  describe('assertPortfolioCustodyAuthorizationValid', () => {
-    test('should not throw with a valid request', () => {
+  describe('PortfolioCustody', () => {
```

Comment:
Why this change?

Reply:
Merge conflict, it used to be bad but I fixed it in this PR, then I wasn't careful resolving the conflict with the it changes.